### PR TITLE
Updating final license acceptance info now that products are released

### PR DESCRIPTION
**Pull request by tyler-ball**
### Description

🚧 Need to wait to merge this until the Chef Workstation 0.4 release goes out

Removes the note necessitated by the fact that we were halfway through our product releases. Also adds info on a product I forgot

### Issues Resolved

N/A

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: chef_master/source/chef_license_accept.rst [hunk 1]
```diff
@@ -213,6 +217,19 @@ Specify the following argument:
 
   knife bootstrap --bootstrap-version 14.12.3
 
+``chef-run``
+-----------------------------------------------------
+``chef-run`` in Chef Workstation >= 0.3 has been updated to add support for accepting the license locally when remotely running Chef Infra Client 15.
+As of Chef Workstation <= 0.4 there is no way to manage the version of Chef Infra Client installed on the remote node.
+It defaults to the latest stable version available.
+
+To accept the license pass the command line flag, set the environment variable or add the following to your ``~/.chef-workstation/config.toml``:
```
Comment (mjingle) [inline chef_master/source/chef_license_accept.rst]: The first part of this sentence does not make sense... 
"To accept the license pass the command line flag,"

Comment (mjingle) [inline chef_master/source/chef_license_accept.rst]:
```suggestion
To accept the license pass the command line flag, set the environment variable or add the following to your ``~/.chef-workstation/config.toml`` file:
```

Comment (tyler-ball) [reply chef_master/source/chef_license_accept.rst]: There are 3 things they can do to accept the license:
1. Pass the `--chef-license` command line flag
2. Set the `CHEF_LICENSE` environment variable
3. Modify their `~/.chef/workstation/config.toml`

What is a better way for me to say that?

Comment (mjingle) [reply chef_master/source/chef_license_accept.rst]: Gotcha! I wasn't parsing that sentence correctly at all, especially when it came to inserting a comma. How about...: 
```suggestion
To accept the license, complete one of the following three tasks. Either pass the ``--chef-license`` command line flag, set the ``CHEF_LICENSE`` environment variable, or add the following to your ``~/.chef-workstation/config.toml`` file:
```

**Review by mjingle — APPROVED** [review]
tenor-650619

**Pull request merged**